db/schema.rb

```diff
@@ -215,19 +215,26 @@
     t.string   "cardholder_name"
     t.string   "card_debit"
     t.string   "card_last_4"
```

Comment:
shouldn't all these payment-method specific fields ship over to the new PaymentMethod table? I think I would expect pretty much everything prefixed with `card_` to be in that table, otherwise people can't e.g. see the last_4 of the payment method they would be deleting